### ISSUE TITLE
v0.8.9 release plan

### ISSUE THREAD

**Issue by thautwarm** [purescript-python/purescripto#8]
I'm now planning to release v0.8.9 after the [new Python FFI providers](https://github.com/purescript-python/purescript-python-ffi-index/pulls) by  @mikesol got registered.

I'd suggest us to focus on reviewing the code of Aff stuffs.

**Comment by thautwarm** [comment]
In parallel, release a new purescript-python compiler solving https://github.com/purescript-python/purescript-python/issues/29 .

**Comment by mikesol** [comment]
Just checking in to see what the release plans are for `0.8.9`? Can I help move it forward?

**Comment by thautwarm** [comment]
Do you have a PyPI account? If no, please create one. I want to share the ownership of this packager with you, then you can decide when to bump up this package.

Initially I want to release purescripto v0.8.9 with a range of compiler-side error message improvements mentioned in purescript-python/purescript-python#29 , but it seems that you have addressed problems in other means. I don't have time to do this so compiler-side changes are deferred.

Now I cannot sync up with all your commits, so I think it must be beneficial if you could come to manage some essential packages instead of only 3rd-party ones.

**Comment by thautwarm** [comment]
Besides, I want to give you owner permission of pspy.

You're quite active in pspy dev, and there seems to be some motivating use cases of pspy in your company, which is an important reason for people to continue some hard tasks.  Besides, I'm not good at making roadmaps, if you can do better in this and gather more active developers, I hope you can take over this kind of management.

**Comment by mikesol** [comment]
Hey! Sorry for the delay. On `pspy` you can add the usernames meeshkan, fornwall, and nzenovkin. All of us take  turns maintaining python repos for our company (ie we're all maintainers of https://pypi.org/project/hmt/). Thanks!

**Comment by thautwarm** [comment]
That's okay! I also have an urgent deadline so could you please invite people yourself?
You are an owner of pspy now, so you can do this.
I searched the user `meeshkan` but find no user with the exact username, so it must be better for you to do this.

**Comment by thautwarm** [comment]
Besides, HMT -- sounds like some practical type system(actually one famous type system takes this name)
